### PR TITLE
init front

### PR DESCRIPTION
**Pull request by SeMech**

### CODE REVIEW

File: CinnabunsFinal/ClientApp/src/main.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+import 'bootstrap-vue/dist/bootstrap-vue.css'
+
+library.add(faPhone, faPaperPlane);
+Vue.component('font-awesome-icon', FontAwesomeIcon);
+Vue.use(BootstrapVue);
+
+Vue.config.productionTip = false;
 
 new Vue({
   router,
```
Comment (BugLight) [inline CinnabunsFinal/ClientApp/src/main.js]: Change ident

File: CinnabunsFinal/ClientApp/src/assets/facebook-messenger-brands.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="facebook-messenger" class="svg-inline--fa fa-facebook-messenger fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="currentColor" d="M224 32C15.9 32-77.5 278 84.6 400.6V480l75.7-42c142.2 39.8 285.4-59.9 285.4-198.7C445.8 124.8 346.5 32 224 32zm23.4 278.1L190 250.5 79.6 311.6l121.1-128.5 57.4 59.6 110.4-61.1-121.1 128.5z"></path></svg>
```
Comment (Kachalov) [inline CinnabunsFinal/ClientApp/src/assets/facebook-messenger-brands.svg:1]: New line

File: CinnabunsFinal/ClientApp/src/components/Header.vue [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            text-align: center;
+            cursor: pointer;
+            padding-top: 15px;
+            font-size: 23px;
+            &:hover {
+                background-color: rgba(255, 202, 191, 0.25);
+            }
+        }
+    }
+</style>
```
Comment (Kachalov) [inline CinnabunsFinal/ClientApp/src/components/Header.vue]: New line

File: CinnabunsFinal/ClientApp/src/views/Events.vue [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    export default {
+        components: {
+            Header: () => import('../components/Header.vue')
+        }
+    }
+</script>
+
+<style lang="scss">
+
+</style>
```
Comment (Kachalov) [inline CinnabunsFinal/ClientApp/src/views/Events.vue]: New line

File: CinnabunsFinal/ClientApp/src/views/Home.vue [hunk 5]
```diff
@@ -1,18 +1,45 @@
...
+    }
+    .title-icon {
+        margin: 40px auto;
+        width: 500px;
+        background-image: url('../assets/logo_federation-01.png');
+        background-size: 100%;
+        background-repeat: no-repeat;
+        height: 150px;
+    }
+</style>
```
Comment (Kachalov) [inline CinnabunsFinal/ClientApp/src/views/Home.vue]: New line

File: CinnabunsFinal/ClientApp/src/views/Partners.vue [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    export default {
+        components: {
+            Header: () => import('../components/Header.vue')
+        }
+    }
+</script>
+
+<style lang="scss">
+
+</style>
```
Comment (Kachalov) [inline CinnabunsFinal/ClientApp/src/views/Partners.vue]: New line

Comment (BugLight) [inline CinnabunsFinal/ClientApp/src/views/Partners.vue]: Add empty lines

File: CinnabunsFinal/ClientApp/src/App.vue [hunk 7]
```diff
@@ -1,29 +1,12 @@
...
-    &.router-link-exact-active {
-      color: #42b983;
+    body {
+        margin: 0;
+        padding: 0;
     }
-  }
-}
-</style>
+</style>
```
Comment (Kachalov) [inline CinnabunsFinal/ClientApp/src/App.vue]: New line

**Review by BugLight — CHANGES_REQUESTED** [review]

**Review by Kachalov — CHANGES_REQUESTED** [review]

**Pull request merged**